indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -404,14 +462,17 @@ export default connect(
         favoriteUsers: userSelectors.getFavoriteUsers(state),
         userFullName: userSelectors.getUserFullName(state),
         availability: state.bookRoom.bookingForm.availability,
+        relatedEvents: state.bookRoom.bookingForm.relatedEvents,
```

Comment:
both this and availability should use selectors.. let's add `getBookingFormAvailability` and `getBookingFormRelatedEvents`?